tools/findFilesThatContainImages.sh

```diff
@@ -0,0 +1,26 @@
+doit(){
+total_count="$(wc -l < "./imgs.txt")";    
+current_count=1;
+spin='←↖↑↗→↘↓↙'
+i=0;
+tput civis
+while read F; do
+    echo "Checking" $F "("$current_count "of" $total_count "image files)";
+    for f in $(find ~/Projects/mongocaleb/docs-realm/source -type f); do 
```

Comment:
Make this relative to the current script path (and not your absolute path!)
```suggestion
    for f in $(find "`dirname $0`"/../source -type f); do 
```